lib/get_it.dart

```
@@ -151,6 +151,11 @@ abstract class GetIt {
   /// If you really need to you can disable the asserts by setting[allowReassignment]= true
   bool allowReassignment = false;
 
+  /// Till V7.6.7 GetIt didn't allow to register multiple instances of the same type.
+  /// if you want to register multiple instances of the same type you can enable this
+  /// and use `getAll()` to retrieve all instances of that parent type
+  static bool allowRegisterMultipleImplementationsOfoneType = false;
```

Is this wholly necessary? What was the behaviour previously? Would it error or fail silently? If it would error, I don't think this is necessary as it wouldn't break/change any already extant behaviour. If the latter, then I guess it's debatable.

I don't generally think it's a good idea to have functionality dictated by flags in properties.